database/connection.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+import (
+	"fmt"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"mocklogin/model"
+)
+
+// global var initialized as a pointer of type gorm.DB, to be able to pass connection to another package
+var DatabaseConnection *gorm.DB
+
+func Connect() {
+	dsn := "user=postgres password=password dbname=postgres port=62626 sslmode=disable"
```

Comment:
can this be an environment variable instead, please? You can do
```suggestion
        user := os.GenEnv("USER")
        password := os.GenEnv("PASSWORD")
        port := os.GenEnv("PORT")
        dbName := os.GenEnv("DB_NAME")
        
	dsn := fmt.Sprintf("user=%s password=%s dbname=%s port=%s sslmode=disable", user, password, dbName, port)
```